dev_guide/builds.adoc

```diff
@@ -1761,6 +1765,130 @@ This example uses `*pullSecret*` in a Source build, but it is also applicable
 in Docker and Custom builds.
 ====
 
+[[build-run-policy]]
+== Build Run Policy
+
+The build run policy describes the order in which the builds created from the
+build configuration should run. This can be done by changing the value of the
+`*runPolicy*` field in the `*spec*` section of the `*Build*` specification.
+
+It is also possible to change the `*runPolicy*` value for existing build
+configurations. 
+
+  * Changing `*Parallel*` to `*Serial*` or `*SerialLatestOnly*` and triggering a
+    new build from this configuration will cause the new build to wait until all
+    parallel builds complete as the "serial" build can only run alone.
+  * Changing `*Serial*` to `*SerialLatestOnly*` and triggering a new build will
+    cause cancellation of all existing builds in queue, except the currently
+    running build. The new build will execute as next.
+
+[[build-serial-run-policy]]
+=== Serial Run Policy
+
+Setting the `*runPolicy*` field to `*Serial*` will cause all new builds created
+from the `*Build*` configuration to be run in "serial" order. That means there
```

Comment:
no scare quotes around serial.  we mean serial literally.
https://en.wikipedia.org/wiki/Scare_quotes